spec/replication/evm_dbsync_spec.rb

```diff
@@ -48,6 +48,15 @@ class ::MasterDb < ActiveRecord::Base; end
     #   based on the configuration
   end
 
+  example ":replicate_backlog" do
```

Comment:
it is :)

i tend to use it for test descriptions that don't read naturally from `it` or `specify` - but happy to change this :)